### PR TITLE
Ss/news feed

### LINKED ISSUE #14: Create news feed page stub

### PR DESCRIPTION
**Pull request by samarseth**
Added news feed stories and basic story structure right now.

PS: The top bar will be removed in the future and story buttons need to be restructured to match the format of the UI design that was presented in the proposal of the project.

Screen Shot 2020-02-20 at 4 31 27 PM

### CODE REVIEW

**Review by pacbac — CHANGES_REQUESTED** [review]
Can you add some screenshots and also fix the merge conflicts?

**Comment by rishabhjain1198** [comment]
We would also want the image in the rectangles in the stories bar to be stretched out to fit.

File: frontend/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -11,6 +11,7 @@
 		3B3967161E833CAA004F5970 /* AppFrameworkInfo.plist in Resources */ = {isa = PBXBuildFile; fileRef = 3B3967151E833CAA004F5970 /* AppFrameworkInfo.plist */; };
```
Comment (rishabhjain1198) [inline frontend/ios/Runner.xcodeproj/project.pbxproj]: What does this file do?

**Comment by rishabhjain1198** [comment]
I'm pretty sure a lot of Xcode project/runner files from frontend/ios should be removed from this PR.

**Comment by goelshashwat** [comment]
@rishabhjain1198 I think stretching the logo in the news stories' button won't look good. It will distort the logo. According to me, what @samarseth did looks good.

**Comment by rishabhjain1198** [comment]
> @rishabhjain1198 I think stretching the logo in the news stories' button won't look good. It will distort the logo. According to me, what @samarseth did looks good.

Fair enough.

**Comment by samarseth** [comment]
> I'm pretty sure a lot of Xcode project/runner files from frontend/ios should be removed from this PR.

Oh I assumed all the files that were supposed to be changed were already in the git ignore. Shouldn't make a dif

**Review by pacbac — APPROVED** [review]

**Pull request merged**